unfoldingWord/translationCore
# Linting

#### This pull request addresses:

Linting is now configured to run before tests.
I've also linted a number of files that have not been touched for at least 3 weeks.

* Rules that **have not** been fully implemented will display warnings.
* Rules that **have** been fully implemented will throw errors

#### How to test this pull request:

make sure all the tests pass!


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2738)
